components/authz-service/storage/v2/postgres/postgres_test.go

```diff
@@ -6941,10 +6962,12 @@ func insertTestRole(t *testing.T,
 	var dbID string
 	require.NoError(t, row.Scan(&dbID))
 
-	_, err := db.Exec(`INSERT INTO iam_role_projects (role_id, project_id)
+	if len(projects) > 0 {
```

Comment:
Does not quite make sense to try an insert when there is nothing to insert, so added the guard clause.